src/azure-cli/azure/cli/command_modules/storage/_params_v2.py

```diff
@@ -0,0 +1,1236 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.profiles import ResourceType
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azure.cli.core.commands.parameters import (tags_type, file_type, get_location_type, get_enum_type,
+                                                get_three_state_flag)
+from azure.cli.core.local_context import LocalContextAttribute, LocalContextAction, ALL
+
+from ._validators import (get_datetime_type, validate_metadata, get_permission_validator, get_permission_help_string,
+                          resource_type_type, services_type, validate_entity, validate_select, validate_blob_type,
+                          validate_included_datasets, validate_custom_domain, validate_container_public_access,
+                          validate_table_payload_format, add_progress_callback, process_resource_group,
+                          storage_account_key_options, process_file_download_namespace, process_metric_update_namespace,
+                          get_char_options_validator, validate_bypass, validate_encryption_source, validate_marker,
+                          validate_storage_data_plane_list, validate_azcopy_upload_destination_url,
+                          validate_azcopy_remove_arguments, as_user_validator, parse_storage_account,
+                          validator_delete_retention_days, validate_delete_retention_days,
+                          validate_fs_public_access)
+
+
+def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statements, too-many-lines
+    from argcomplete.completers import FilesCompleter
+    from six import u as unicode_string
+
+    from knack.arguments import ignore_type, CLIArgumentType
+
+    from azure.cli.core.commands.parameters import get_resource_name_completion_list
+
+    from .sdkutil import get_table_data_type
+    from .completers import get_storage_name_completion_list
+
+    t_base_blob_service = self.get_sdk('blob.baseblobservice#BaseBlobService')
+    t_file_service = self.get_sdk('file#FileService')
+    t_queue_service = self.get_sdk('queue#QueueService')
+    t_table_service = get_table_data_type(self.cli_ctx, 'table', 'TableService')
```

Comment:
The definition of these xxxServices comes from track 1, right? Is there any way to reduce the dependence of track 1 they introduce on track 2?

Reply:
chat offline already.